server/controllers/stock/inventory/groups.js

```diff
@@ -77,6 +78,23 @@ function getGroups(uid) {
   return db.exec(sql, [uid]);
 }
 
+/**
+ * Count inventory in the group
+ * @param {string} uid the group uuid
+ */
+function countInventory(uid) {
+  'use strict';
+
+  uid = (uid) ? db.bid(uid) : undefined;
```

Comment:
This check is not necessary.  For this function to be called, the path `/inventory/groups/:uuid/count` will have been matched.  That means that `:uuid` will have been defined.